packages/core/src/transport/sendWithRetryStrategy.ts

```diff
@@ -101,6 +101,10 @@ function send(
       // do not consider transport down if another ongoing request could succeed
       state.transportStatus =
         state.bandwidthMonitor.ongoingRequestCount > 0 ? TransportStatus.FAILURE_DETECTED : TransportStatus.DOWN
+      payload.retry = {
+        count: payload.retry ? (payload.retry.count += 1) : 1,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
        count: payload.retry ? (payload.retry.count + 1) : 1,
```